scripts/feature_tests/upgrade/README.md

```diff
@@ -0,0 +1,22 @@
+# Upgrade script naming scheme
+
+* nodes: ```<i>cp | <j>w; i=1..n, j=0..n```
+* what: bootDiskImage | k8sVer | k8sBin
+* How: scaleInWorkers | scaleOutWorkers
+* other: extraNode
+* postfix: upgrade | upgrade_both; cp or w depending on the script directory
+ or both cp and w
```

Comment:
Could you maybe mention also that cp stands for controlplane, and w stands for worker ? It is not clear for someone that is not familiar with those short names.

Reply:
Done and this should be merged first.